### ISSUE TITLE
Document plot_partials plotting keyword

### ISSUE THREAD

**Issue by asher-pembroke** [EnsembleGovServices/kamodo-core#77]
## Describe the bug
The documentation for usage of `plot_partial` keyword is nonexistent. https://ensemblegovservices.github.io/kamodo-core/API/#multi-function-plots

The following example needs to be added to the documentation somewhere to explain the use case of plotting a lower dimensional slice of a high-dimensional function.

```python
from kamodo import kamodofy, gridify, Kamodo
from scipy.interpolate import RegularGridInterpolator
import numpy as np

#define sample coordinate and data arrays
t, lon, lat, ht = np.linspace(0.,24.,10), np.linspace(0.,360.,20), np.linspace(-90.,90.,50), np.linspace(100.,10000.,250)
variable = np.reshape(np.linspace(0.,2 * np.pi,10 * 20 * 50 * 250), (10,20,50,250))

#define and kamodofy interpolating function
rgi = RegularGridInterpolator((t, lon, lat, ht), variable, bounds_error = False, fill_value=np.NaN)

@kamodofy(units='m/s', data=variable)
def interpolator(xvec):
    """Interpolates 4d variable without a grid"""
    return rgi(xvec)

#gridify same function
interpolator_grid = kamodofy(gridify(interpolator, time = t, lon=lon, lat = lat,  height = ht), units='m/s', data=variable, arg_units={'time':'hr','lon':'deg','lat':'deg','height':'km'})

#register in a new kamodo object
kamodo_object = Kamodo()
kamodo_object['v'] = interpolator
kamodo_object['v_ijkl'] = interpolator_grid # 4 dimensional
kamodo_object

kamodo_object.plot(v_ijkl = {'time':10.,'lat':90.})  #output: not supported: out_dim ('N', 'M'), arg_dims [(1,), ('N',), (1,), ('M',)]
```

What's happening here is that kamodo doesn't know how to plot higher dimensional objects (humans live in 3D and technically only see in 2D). To work around this, we can use the `plot_partial` keyword to lower the dimensionality of a function by fixing some of its variables, like so:

```python
kamodo_object.plot('v_ijkl', plot_partial={'v_ijkl': {'lon':10.,'lat':90.}})
```

![newplot (9)](https://user-images.githubusercontent.com/13787870/162788770-eea828b9-ea17-4716-b66b-9f8b00446fdd.png)

**Comment by iamjavaexpert** [comment]
I am getting this error when i run mkdocs serve or mkdocs build in plotting branch. 
![Screenshot from 2022-04-14 16-54-08](https://user-images.githubusercontent.com/70366572/163382406-2f6ecce0-d8a3-4344-a3f0-ea83ace417f5.png)

**Comment by iamjavaexpert** [comment]
@asher-pembroke Added documentation for plot_partial keyword usage as described above.

**Comment by asher-pembroke** [comment]
Fixed with #80
